charts/tidb-operator/templates/NOTES.txt

```diff
@@ -1,7 +1,2 @@
-1. Make sure tidb-operator components are running
-   kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/instance={{ .Release.Name }}
-2. Install CRD
```

Comment:
When installing tidb-operator, CRD has already been installed in [the](https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-tidb-operator/#install-tidb-operator) [documents](https://pingcap.com/docs/tidb-in-kubernetes/stable/get-started/#deploy-tidb-operator).